var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -144,14 +158,8 @@ def install(self, spec, prefix):
 
     @property
     def libs(self):
-        libs = find_libraries('libcudart', root=self.prefix, shared=True,
-                              recursive=True)
-
-        filtered_libs = []
-        # CUDA 10.0 provides Compatability libraries for running newer versions
-        # of CUDA with older drivers. These do not work with newer drivers.
-        for lib in libs:
-            parts = lib.split(os.sep)
-            if 'compat' not in parts and 'stubs' not in parts:
```

Comment:
> What do we do with this PR?

I think the only thing needed for now it to restore the filtering of 'stubs' and 'compat' from `libs` (`driver_libs` should keep 'stubs' though as discussed before in this thread).